Automattic/jetpack
# General: update WordPress version requirements to WP 5.6

#### Changes proposed in this Pull Request:

WordPress 5.7 will be released on March 9, 2021, and from that point on Jetpack will require WP 5.6.

This PR introduces the following changes:

- Update our "tested up to" version
- Update the version of WordPress required for Jetpack from WP 5.5 to WP 5.6
- Changes to Toolbar components ([Reference](https://make.wordpress.org/core/2020/11/18/changes-to-toolbar-components-in-wordpress-5-6/))
- Move away from `IconButton` ([Reference](https://make.wordpress.org/core/2020/02/13/an-updated-button-component-in-wordpress-5-4/))
- Move from `formattingControls` to `allowedFormats` ([Reference](https://github.com/WordPress/gutenberg/blob/master/packages/block-editor/CHANGELOG.md#300-2019-08-05))
- Other block deprecations

#### Jetpack product discussion

Primary issue: #18842
#### Does this pull request change what data or activity we track or use?

* No

#### Testing instructions:

* Do the tests pass?
* Test all the blocks that were updated in this PR (see labels), and ensure they still work. You'll want to pay extra attention to the block toolbars that appear above each block.

#### Proposed changelog entry for your changes:

* General: Jetpack now requires WordPress 5.6, and is fully compatible with WordPress 5.7.
